### PR TITLE
Add CRUD!

### PR DESCRIPTION
**Pull request by Banstein**
implement the CRUD (create, update, delete) methods :

- Implement a function for adding a new task.
- Implement a function for deleting a task.
- Implement a function for editing task descriptions.
- Update the item's indexes and the local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,41 +1,113 @@
...
+    const editer = prompt('Update Task!');
+    const eLi = e.target.parentNode.parentNode.parentNode;
+    list.forEach((item) => {
+      if (editer.length > 1 && item.index === Number(eLi.id)) {
+        item.description = editer;
+      }
+    });
+    UpdateStorage(list);
+    displayTasks(list);
+  }
```
Comment (ZeenatLawal) [inline src/index.js:108]: - [ ] The edit task functionality should be implemented as shown in the video [demo](https://www.youtube.com/watch?v=AcUd-_Yjjqg). You can implement this by changing the textarea to an input field on click of the edit button or on a double click of the textarea. Kindly modify this.

Comment (ZeenatLawal) [inline src/index.js:108]: - [ ] All the new functionalities(add, delete and edit) should be placed in a new JavaScript file as stated in the requirements.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @Banstein, 

Great job so far!:clap: :clap:
## To Highlight 🎊 🎊 
- GitHub flow is followed. ✔️
- PR has a title and summary. ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bellom** [comment]
Hello CR,

Kindly go ahead and approve this PR.

Thank you.

CR Leader.

**Pull request merged**